tsrc/github.py

```diff
@@ -11,6 +11,9 @@
 import tsrc
 
 
+DEFAULT_GITHUB_AUTH_SYSTEM = "github"
```

Comment:
Not sure why you are using a constant for the 'github' string and not for 'github'

I think you could just drop this constant, it's one less indirection when reading the source code